dasharo-compatibility/wifi-bluetooth-support.robot

```diff
@@ -35,20 +39,20 @@ WLE001.001 Wireless card detection (Ubuntu 20.04)
     Login To Linux
     Switch To Root User
     ${out}=    Execute Command In Terminal    lspci | grep "Network controller:"
-    Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
+    Should Contain    ${out}    *${WIFI_CARD_UBUNTU}*
     Exit From Root User
 
-WLE001.002 Wireless card detection (Windows 11)
+WLE001.002 Wireless card detection (Windows 11) (Intel)
```

Comment:
You shouldn't change it.
For protectli-v1*10 the variable values are:
```
${WIFI_CARD}=                                       Qualcomm
${WIFI_CARD_UBUNTU}=                                Qualcomm
```

Reply:
[Names](https://github.com/Dasharo/open-source-firmware-validation/pull/136/files#diff-db99b3a0adebbc880d28cd1f87d49ea87c9b11cd1d3681f599bd1494c26e814cL41) no longer changed